### PR TITLE
MON-3792: fix metrics-server path for /test versions

### PR DESCRIPTION
**Pull request by rexagod**
Owing to the older path being referenced in the prow workflow, we saw consistent failure for /test versions job, this fixes that.

***

* [ ] I added CHANGELOG entry for this change.
* [x] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

File: hack/go/generate_versions.go [hunk 1]
```diff
@@ -199,20 +204,50 @@ func getVersion(repo, ref string) (string, error) {
...
+	var (
+		version string
+		err     error
+	)
+	versionChan := make(chan string, 1)
+	errChan := make(chan error, 1)
+	ctx, cancel := context.WithCancel(context.Background())
+
+	for _, path := range paths {
+		go func(path string, cancel context.CancelFunc) {
```
Comment (machine424) [inline hack/go/generate_versions.go:218]: No need for concurrency here I think, no problem if it takes an extra second.
I'd run it sequentially to keep the code simple/more readable.
/lgtm
so we can move forward, but I'll
/hold
please don't hesitate to unhold.

Comment (rexagod) [reply hack/go/generate_versions.go]: Removed.

File: Documentation/deps-versions.md [hunk 2]
```diff
@@ -1,10 +0,0 @@
-|  OCP Version |                                   alertmanager                                   |                              kubeRbacProxy                               |                              kubeStateMetrics                               |                              kubernetesMetricsServer                              |                             monitoringPlugin                              |                             nodeExporter                              |                              promLabelProxy                              |                             prometheus                              |                                prometheusAdapter                                |                              prometheusOperator                              |                             thanos                              |
```
Comment (machine424) [inline Documentation/deps-versions.md:1]: I'm afraid, it generates an empty table now

**Comment by machine424** [comment]
Thanks
/lgtm

**Pull request merged**